web/app/js/components/TopModule.jsx

```diff
@@ -103,7 +103,7 @@ class TopModule extends React.Component {
   }
 
   onWebsocketClose = e => {
```

Comment:
I think we may want to update this method to be

```
  onWebsocketClose = e => {
    this.updateTapClosingState();
    /* We ignore any abnormal closure since it doesn't matter as long as
    the connection to the websocket is closed. This is also a workaround
    where Chrome browsers incorrectly displays a 1006 close code
    https://github.com/linkerd/linkerd2/issues/1630
    */
    if (e.code !== WS_NORMAL_CLOSURE && e.code !== WS_ABNORMAL_CLOSURE) {
      this.setState({
        error: {
          error: `Websocket close error [${e.code}: ${wsCloseCodes[e.code]}] ${e.reason ? ":" : ""} ${e.reason}`
        }
      });
    }
  }
```
We'll need to add these consts near the top of the file:
```
const WS_NORMAL_CLOSURE = 1000;
const WS_ABNORMAL_CLOSURE = 1006;
```

That will allow us to pass any errors on to the default banner handling, which I think is better than nothing until we implement the contextual banner discussed in #2461